src/Spatial/Euclidean/Circle3D.cs

```diff
@@ -16,13 +16,50 @@ public Circle3D(Point3D centerPoint, UnitVector3D axis, double radius)
             this.Radius = radius;
         }
 
+        /// <summary>
+        /// Create a circle from the midpoint between two points, in a direction along a specified axis
+        /// </summary>
+        /// <param name="p1">First point on the circumference of the circle</param>
+        /// <param name="p2">Second point on the circumference of the circle</param>
+        /// <param name="axis">Direction of the plane in which the circle lies</param>
         public Circle3D(Point3D p1, Point3D p2, UnitVector3D axis)
         {
             this.CenterPoint = Point3D.MidPoint(p1, p2);
             this.Axis = axis;
             this.Radius = p1.DistanceTo(CenterPoint);
         }
 
+        /// <summary>
+        /// Create a circle from three points which lie along its circumference.
+        /// </summary>
+        /// <param name="p1"></param>
+        /// <param name="p2"></param>
+        /// <param name="p3"></param>
+        public Circle3D(Point3D p1, Point3D p2, Point3D p3)
+        {
+            https://www.physicsforums.com/threads/equation-of-a-circle-through-3-points-in-3d-space.173847/
```

Comment:
Maybe some input validation here? If they are all the same point or points on a line?